emscripten.py

```diff
@@ -2152,9 +2152,7 @@ def main(args, compiler_engine, cache, temp_files, DEBUG):
   for setting in args.settings:
     name, value = setting.strip().split('=', 1)
     value = json.loads(value)
-    if isinstance(value, unicode):
```

Comment:
can't we use `asstr` in these places?

I admit I am getting confused with `asstr, asbytes` and others. Perhaps we should write up a Python2/3 developer's guide, explaining the proper best practices?

Reply:
>can't we use asstr in these places?

We can, thus the line 2155?

Reply:
Oh sorry, I completely misread this diff... :)